redhat-cop/infra-ansible
# Uploading LDAP CA cert to Ansible Tower on OCP

### What does this PR do?
This PR adds a helper role for config-ansible-tower-ocp that's providing capability of setting custom CA for LDAP SSL.

### How should this be tested?
After test run, verify if /etc/cert/ldap.pem is present in the Ansible Tower Web and Ansible Tower Task containers

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
